package.json

```diff
@@ -40,13 +39,13 @@
     "babel-preset-es2015": "^7.0.0-beta.3",
     "babel-preset-react": "^7.0.0-beta.3",
     "babel-preset-stage-2": "^7.0.0-beta.3",
+    "react": "^>=16.3 < 18",
+    "react-dom": "^>=16.3 < 18",
```

Comment:
For `devDependencies`, you can just use the specific version: `^16.3.0`. The range is for the consuming app.